### PR TITLE
India: Update locality polygons

### LINKED ISSUE #1589: Add locality polygons to the largest cities in India
Add/update polygons for the ~100 largest cities in India.

### PR DESCRIPTION
**Pull request by stepps00**
Fixes #1589.

Includes:
- [x] Updated geometries for localities in India (102 of them) with existing geometries stored in alt files
- [x] New locality records (just one for Navi Mumbai)
- [x] Updated existing locality records that fall within the new geometries with `mz:is_funky` of 1
- [x] PIP work
- [x] Update `wof:name` and `name:eng_x_preferred` values

This will require a follow-up issue and remapping the changes into the `whosonfirst-data-admin`in` repo.

### CODE REVIEW

**Comment by stepps00** [comment]
Commit Notes:

e0b61bc - updated existing locality records in India, stored existing geometries as alt geometries
96bb915 - created a single new locality record for Navi Mumbai
a52cada - flags all locality records with point geometries with `mz:is_funky` = 1
b917902 - flags all locality records with polygon geometries with `mz:is_funky` = 1

File: data/859/281/29/85928129.geojson [hunk 1]
```diff
@@ -75,21 +69,20 @@
     "wof:geomhash":"9f802422b5181a63e863c4d0a3530639",
     "wof:hierarchy":[
         {
-            "continent_id":102191569,
-            "country_id":85632469,
-            "county_id":890507423,
-            "locality_id":102030425,
+            "continent_id":-1,
```
Comment (nvkelso) [inline data/859/281/29/85928129.geojson]: This is surprising, is it expected?

image

File: data/859/306/73/85930673.geojson [hunk 2]
```diff
@@ -83,8 +84,16 @@
         {
             "continent_id":102191569,
             "country_id":85632469,
-            "county_id":890507427,
-            "locality_id":102028703,
+            "county_id":1108696315,
+            "locality_id":102030059,
+            "neighbourhood_id":85930673,
+            "region_id":85672141
```
Comment (nvkelso) [inline data/859/306/73/85930673.geojson]: So the region geometry overlaps here? Please file followup issue.

File: data/859/338/67/85933867.geojson [hunk 3]
```diff
@@ -89,7 +90,7 @@
             "continent_id":102191569,
             "country_id":85632469,
             "county_id":890508847,
-            "locality_id":-1,
+            "locality_id":102030955,
```
Comment (nvkelso) [inline data/859/338/67/85933867.geojson]: Win!

File: data/890/506/073/890506073.geojson [hunk 4]
```diff
@@ -46,16 +46,16 @@
...
             "neighbourhood_id":890506073,
             "region_id":85672123
         }
     ],
     "wof:id":890506073,
-    "wof:lastmodified":1527782053,
+    "wof:lastmodified":1555547788,
     "wof:name":"Dilli Haat",
-    "wof:parent_id":102030425,
+    "wof:parent_id":-1,
```
Comment (nvkelso) [inline data/890/506/073/890506073.geojson]: Why did it pick up a new neighbourhood, but not a new parent?

File: data/859/281/29/85928129.geojson (hunk 1 above)
Comment (stepps00) [reply data/859/281/29/85928129.geojson]: Hmm, that is surprising. I re-PIP'd the records and see no difference in the branch. A few other neighbourhoods also lost some hierarchy information, also unexpected.

I will track this in a follow up.

File: data/890/506/073/890506073.geojson (hunk 4 above)
Comment (stepps00) [reply data/890/506/073/890506073.geojson]: Also unexpected, will track in a follow-up.

**Pull request merged**